Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ConfigurationHistory/ConfigurationHistory.tsx

```diff
@@ -114,17 +120,10 @@ function ConfigurationHistory(props: { MeterConfigurationID: number, MeterKey: s
 
 
             <hr />
-            <ul className="nav nav-tabs">
-                <li className="nav-item">
-                    <a className={"nav-link" + (tab == "configuration" ? " active" : "")} onClick={() => setTab('configuration')} data-toggle="tab" href="#configuration">Configuration</a>
-                </li>
-                <li className="nav-item">
-                    <a className={"nav-link" + (tab == "filesProcessed" ? " active" : "")} onClick={() => setTab('filesProcessed')} data-toggle="tab" href="#filesProcessed">Files Processed</a>
-                </li>
-            </ul>
+            <TabSelector CurrentTab={Tab} SetTab={setTab} Tabs={Tabs} />
 
             <div className="tab-content" style={{ maxHeight: window.innerHeight - 235, overflow: 'hidden' }}>
-                <div className={"tab-pane " + (tab == "configuration" ? " active" : "fade")} id="configuration">
+                <div className={"tab-pane " + (Tab == "configuration" ? " active" : "fade")} id="configuration">
```

Comment:
```suggestion
                {Tab=== 'configuration'? <div className={"tab-pane active"} >
```
couple items for tabs:
(1) remove the `id` unless it is actually needed
(2) Turns out if we always render the tabs (even those not selected) and just adjust the class we will have issues with reloading items. Let's switch to only rendering the tab that is actually selected and make the class `active`